include/mbedtls/md.h

```diff
@@ -151,7 +151,16 @@ typedef enum {
     MBEDTLS_MD_RIPEMD160, /**< The RIPEMD-160 message digest. */
 } mbedtls_md_type_t;
 
-/* Note: this should be kept in sync with PSA_HASH_MAX_SIZE */
+/* Note: this should always be >= PSA_HASH_MAX_SIZE
+ * in all builds with both CRYPTO_C and MD_LIGHT.
```

Comment:
I'd like to validate this. Can we please add this somewhere that has both macros available, maybe `md.c`?
```
#if defined(MBEDTLS_PSA_CRYPTO_CLIENT) && MBEDTLS_MD_MAX_SIZE < PSA_HASH_MAX_SIZE
#error ...
#endif
```

Reply:
I went with `MBEDTLS_PSA_CRYPTO_C` as that's what guards the `#include <psa/crypto.h>` currently.